seastar.yaml

```diff
@@ -0,0 +1,244 @@
+package:
+  name: seastar
+  epoch: 0
+  version: "25.05.0"
+  description: "Seastar is a high-performance, open-source framework for building C++ applications"
+  copyright:
+    - license: "Apache-2.0"
+  dependencies:
+    runtime:
+      - binutils
+      - boost-dev
+      - c-ares-dev
+      - fmt-dev
+      - gcc
+      - pkgconf
+      - protobuf-dev
+
+environment:
+  contents:
+    packages:
+      - abseil-cpp-dev
+      - bison
+      - boost-dev
+      - build-base
+      - c-ares-dev
+      - doxygen
+      - fmt-dev
+      - gnutls-dev
+      - gtest-dev
+      - hwloc-dev
+      - libsctp-dev
+      - liburing-dev
+      - lz4-dev
+      - numactl-dev
+      - pkgconf-dev
+      - protobuf-dev
+      - python-${{vars.py-version}}
+      - ragel
+      - util-linux-dev
+      - valgrind-dev
+      - xfsprogs-dev
+      - yaml-cpp-dev
+
+vars:
+  py-version: 3.13
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: "https://github.com/scylladb/seastar.git"
+      tag: seastar-${{package.version}}
+      expected-commit: 8df8212e53577e1d8477a5c901457cd61d88afc7
+
+  - uses: cmake/configure
+    with:
+      opts: |
+        -DSeastar_TESTING=OFF \
+        -DPROTOBUF_SOURCE=SYSTEM \
+        -DBUILD_SHARED_LIBS=ON \
+        -DWITH_SSL=system
```

Comment:
Should we pass `-DCMAKE_BUILD_TYPE=RelWithDebInfo` which sets `-O3`?

Reply:
Same as below, I expect that building a dynamic lib is a better approach here as we don't have to pin versions.